### PR TITLE
Added fireaxe cabinets

### PR DESCRIPTION
**Pull request by SweptWasTaken**
## About the PR 

Not gonna map these in until SecureEntityStorage is refactored and I can add locks to this. Likewise i'm not going to put it in the changelog because this won't be player-facing until it's mapped in.

I was thinking about changing the ItemCabinetVisualizer to be layer-based instead of state-based. I'm also looking at #4201 and know that some cabinets have different fill sprites when containing different objects.

Other than that these are functional.

### CODE REVIEW

File: Resources/Prototypes/Entities/Constructible/Walls/fireaxe_cabinet.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  name: fire axe cabinet
+  description: There is a small label that reads \"For Emergency use only\" along with details for safe use of the axe. As if.
+  components:
+    - type: Clickable
+    - type: InteractionOutline
+    - type: Sprite
+      sprite: Constructible/Misc/fireaxe_cabinet.rsi
+      state: cabinet-filled-closed
+    - type: ItemCabinet
+      doorSound: /Audio/Machines/machine_switch.ogg
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Constructible/Walls/fireaxe_cabinet.yml:12]: netsync false?

**Review by metalgearsloth — APPROVED** [review]

File: Resources/Prototypes/Entities/Constructible/Walls/fireaxe_cabinet.yml (hunk 1 above)
Comment (SweptWasTaken) [reply Resources/Prototypes/Entities/Constructible/Walls/fireaxe_cabinet.yml:12]: Sure i'll update extinguisher cabinet as well with this.

File: Resources/Textures/Constructible/Storage/Cabinets/extinguisher_cabinet.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      "name": "extinguisher",
+    },
+    {
+      "name": "extinguisher-mini",
+    },
+    {
+      "name": "frame",
+    },
+    {
+      "name": "open",
```
Comment (metalgearsloth) [inline Resources/Textures/Constructible/Storage/Cabinets/extinguisher_cabinet.rsi/meta.json:35]:
```suggestion
      "name": "open"
```

File: Resources/Textures/Constructible/Storage/Cabinets/extinguisher_cabinet.rsi/meta.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      "name": "closed",
+    },
+    {
+      "name": "extinguisher",
+    },
+    {
+      "name": "extinguisher-mini",
+    },
+    {
+      "name": "frame",
```
Comment (metalgearsloth) [inline Resources/Textures/Constructible/Storage/Cabinets/extinguisher_cabinet.rsi/meta.json:32]:
```suggestion
      "name": "extinguisher_closed"
    },
    {
      "name": "extinguisher_empty"
    },
    {
      "name": "extinguisher_full"
    },
    {
      "name": "extinguisher_mini"
    },
    {
      "name": "closed"
    },
    {
      "name": "extinguisher"
    },
    {
      "name": "extinguisher-mini"
    },
    {
      "name": "frame"
```

**Pull request merged**